js/app.jsx

```diff
@@ -319,32 +319,50 @@ var CaseverList = React.createClass({
     }.bind(this));
   },
 
+  getInitialState: function() {
+    return ({totalCount: 0});
+  },
+
+  componentDidUpdate: function() {
+    if(this.state.totalCount!=this.props.casevers.meta.total_count) {
+      this.setState({totalCount: this.props.casevers.meta.total_count});
+    }
+  },
+
   render: function() {
     //can use the casevers.meta
     var casevers = this.props.casevers.objects.map(function(casever){
       return (<CaseverListItem casever={casever} onChange={this.props.handleCheck} handleAddFilter={this.props.handleAddFilter}/>)
     }.bind(this))
 
-    return (
-      <Row>
-      <Table striped condensed hover className="caseverList">
-        <tbody>
-          <tr>
-            <th><input type="checkbox" ref="checkAllBox" onChange={this.checkAll}/></th>
-            <th>ID</th>
-            <th>status</th>
-            <SortableTh name="name" filter="name" handleAddFilter={this.props.handleAddFilter}></SortableTh>
-            <SortableTh name="priority" filter="case__priority" handleAddFilter={this.props.handleAddFilter}></SortableTh>
-            <SortableTh name="product" filter="productversion" handleAddFilter={this.props.handleAddFilter}></SortableTh>
-            <SortableTh name="modified" filter="modified_on" handleAddFilter={this.props.handleAddFilter}></SortableTh>
-            <th></th>
-            <th></th>
-          </tr>
-          {casevers}
-        </tbody>
-      </Table>
-      </Row>
-    )
+    if (this.state.totalCount>0) {
+      return (
+          <Row>
+            <Table striped condensed hover className="caseverList">
+              <tbody>
+              <tr>
+                <td colSpan="9">Total {this.state.totalCount} suites found</td>
```

Comment:
What if we use `this.props.suites.meta.total_count` directly, without saving it in the state first?